src/platform/mhv/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@department-of-veterans-affairs/mhv",
+  "version": "1.0.0",
+  "scripts": {
+    "demo": "echo DEMOING $npm_package_name v $npm_package_version with WORKSPACES"
+  },
+  "exports": {
+    ".": "./exportsFile.js"
```

Comment:
I can't comment on this export file, but consider adding a comment in there saying something like intentionally left blank, or what the file will contain.

Reply:
Yes, intentionally left blank for the moment.